### ISSUE TITLE
ng g @co-it/schematics:jest

### ISSUE THREAD

**Issue by GregOnNet** [co-IT/schematics#4]
## Jest
> [briebug/jest-schematic](https://github.com/briebug/jest-schematic) can serve as a template

- [x] Download jest-preset-angular
- [x] Download jest
- [x] Create `jest.config.js` in root directory
- [x] Create `setup-jest.ts` in `src/`

## Karma

- [x] Remove karma configuration
- [x] Remove karma dependencies from `package.json`

### Protractor

- [ ] jasmine dependencies cannot be removed since pratractor is already in use
- [ ] Check if `@types/jasmine` & `@types/jest` can be used in parallel
       - Maybe update `types` setting in `tsconfig.spec.json` & `tsconfig.e2e.json` to avoid conflicts
       - jasmine for protractor
       - jest for app

**Comment by GregOnNet** [comment]
I updated the description.
I do not think that we should delete all jasmine dependencies on our own.
`protractor` could be used.

We need to configure the Angular Project, that jasmine can be used of e2e tests and jest for unit and integration tests.

**Comment by GregOnNet** [comment]
How is your plan to implement the jest schematic?
I want to release the first version this week.
Therefore I want to have jest in place.

I can help to get this done if you want.

**Comment by Strulle** [comment]
Multiple issues found.
[ ] remove test.ts
[ ] configure tsconfig.spec.ts
[ ] remove "test"-architect form angular.json

**Comment by Strulle** [comment]
Multiple issues found.
[ ] remove test.ts
[ ] configure tsconfig.spec.ts
[ ] remove "test"-architect form angular.json

**Comment by GregOnNet** [comment]
@Strulle, @Markus-Ende 	We may need a final discussion if we want to switch to [@angular-builders/jest](https://github.com/meltedspark/angular-builders/tree/master/packages/jest).

I would align with cypress since we also use a builder there.

**Comment by GregOnNet** [comment]
@Strulle If you are done with this issue you can close it.
Please open a new issue where we can discuss if we introduce `@angular-builders/jest`

**Comment by GregOnNet** [comment]
Closing this issue in favour of #33
